src/azure-cli/azure/cli/command_modules/storage/_format.py

```diff
@@ -4,38 +4,13 @@
 # --------------------------------------------------------------------------------------------
 """Table transformer for storage commands"""
 
+from azure.cli.core.commands.transform import build_table_output
 from azure.cli.core.profiles import get_sdk, ResourceType
 from knack.log import get_logger
 
 logger = get_logger(__name__)
 
 
-def build_table_output(result, projection):
```

Comment:
then we also need to change storage module code using `build_table_output` imported from `src/azure-cli/azure/cli/command_modules/storage/_format.py`.

Reply:
Exactly, you can see Line7~